### PR TITLE
FEATURE: [bitget] integrate QueryMarkets, QueryTicker and QueryAccount api

### PR DESCRIPTION
**Pull request by c9s**

### CODE REVIEW

File: pkg/exchange/bitget/exchange.go [hunk 1]
```diff
@@ -43,3 +47,80 @@ func (e *Exchange) Name() types.ExchangeName {
...
 	return PlatformToken
 }
+
+func (e *Exchange) NewStream() types.Stream {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
+	// TODO implement me
```
Comment (bailantaotao) [inline pkg/exchange/bitget/exchange.go:57]: rm comments

File: pkg/exchange/bitget/exchange.go [hunk 2]
```diff
@@ -43,3 +47,80 @@ func (e *Exchange) Name() types.ExchangeName {
...
+	req := e.client.NewGetSymbolsRequest()
```
Comment (bailantaotao) [inline pkg/exchange/bitget/exchange.go:58]: add rate limiter?

File: pkg/exchange/bitget/exchange.go [hunk 3]
```diff
@@ -86,8 +86,23 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
 }
 
 func (e *Exchange) QueryTicker(ctx context.Context, symbol string) (*types.Ticker, error) {
-	// TODO implement me
-	panic("implement me")
+	req := e.client.NewGetTickerRequest()
```
Comment (bailantaotao) [inline pkg/exchange/bitget/exchange.go:89]: rate limiter?

File: pkg/exchange/bitget/exchange.go [hunk 4]
```diff
@@ -116,8 +116,21 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
 }
 
 func (e *Exchange) QueryAccount(ctx context.Context) (*types.Account, error) {
-	// TODO implement me
-	panic("implement me")
+	req := e.client.NewGetAccountAssetsRequest()
```
Comment (bailantaotao) [inline pkg/exchange/bitget/exchange.go:119]: rate limiter?

File: pkg/exchange/bitget/exchange.go [hunk 5]
```diff
@@ -116,8 +116,21 @@ func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval type
...
+	resp, err := req.Do(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	bals := types.BalanceMap{}
+	for _, asset := range resp {
+		b := toGlobalBalance(asset)
+		bals[asset.CoinName] = b
+	}
```
Comment (bailantaotao) [inline pkg/exchange/bitget/exchange.go:129]: Perhaps you can move this implementation to QueryAccountBalances, and then QueryAccount can utilize it.

**Review by bailantaotao — APPROVED** [review]
Everything else LGTM

File: pkg/exchange/bitget/exchange.go (hunk 1 above)
Comment (zenixls2) [reply pkg/exchange/bitget/exchange.go:57]: Or write "TODO: " for color highlight

File: pkg/exchange/bitget/exchange.go [hunk 6]
```diff
@@ -43,3 +47,108 @@ func (e *Exchange) Name() types.ExchangeName {
...
+}
+
+func (e *Exchange) QueryTickers(ctx context.Context, symbol ...string) (map[string]types.Ticker, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval types.Interval, options types.KLineQueryOptions) ([]types.KLine, error) {
+	// TODO implement me
+	panic("implement me")
```
Comment (zenixls2) [inline pkg/exchange/bitget/exchange.go:115]: Please mark in document that bitget is not ready for backtesting

Comment (zenixls2) [reply pkg/exchange/bitget/exchange.go:115]: Just suggestion. Users should also be able to find this on their own

**Review by austin362667 — APPROVED** [review]

**Pull request merged**